app/templates/components/forms/session-speaker-form.hbs

```diff
@@ -84,7 +84,7 @@
       </div>
       <div class="menu">
```

Comment:
@mohitm15 Do this and get it reviewed by @CosmicCoder96 
 
```hbs

      <div class="menu">
        {{#each speakers as |speaker|}}
          {{#if speaker.name}}
            <div data-value="{{map-value mapper speaker}}" class="item">
              {{speaker.name}}
            </div>
          {{/if}}
        {{/each}}
      </div>

```

Reply:
> @mohitm15 Do this and get it reviewed by @CosmicCoder96
> 
> ```
>       <div class="menu">
>         {{#each speakers as |speaker|}}
>           {{#if speaker.name}}
>             <div data-value="{{map-value mapper speaker}}" class="item">
>               {{speaker.name}}
>             </div>
>           {{/if}}
>         {{/each}}
>       </div>
> ```

That is working @uds5501 👍

Reply:
@mohitm15 Can you just add this commit and get it reviewed? The code is right here, presented to you on a plate, what's the latency for?

Reply:
@mohitm15 Still present ?